### PR TITLE
Cleanup strings

### PR DESCRIPTION
**Pull request by Lineflyer**
- Work on #10155
- Remove strings, which are reported unused by lint ( @cgeo/team Please doublecheck, as I don't know how true these lint reports are)
- Replace unmatching variables (decimal instead of string)

### CODE REVIEW

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -1839,7 +1831,7 @@
 
     <plurals name="days_ago">
         <item quantity="zero">%d days ago</item>
-        <item quantity="one">yesterday</item>
+        <item quantity="one">%d day ago</item>
```
Comment (fm-sys) [inline main/res/values/strings.xml:1834]: Why don't keep yesterday?

Comment (fm-sys) [reply main/res/values/strings.xml:1834]: Ignore me, found the reason in https://github.com/cgeo/cgeo/issues/10155#issue-831002685

**Comment by Lineflyer** [comment]
@eddiemuc or @bekuno 

Does this look OK for you?
It should tackle some lint issues about missing quantities (once translations come in) and unused resources.
But please check if the resources are really unused.

**Comment by eddiemuc** [comment]
@Lineflyer strings in context of SAF-stuff look ok to remove

**Pull request merged**